requirements-optional.txt

```diff
@@ -12,3 +12,4 @@ tesserocr>=1.2; platform_system != 'Windows'
 akismet>=1.0
 boto3
 zeep
+social-auth-core[openidconnect]>=3.1.0
```

Comment:
Thinking about it more, this makes it harder to maintain - it's yet another place where version needs to be adjusted. So probably it's better to not list this at all.

Reply:
OK, but how can a user add this then? And how does he/she knows, if using a provider from social_auth that uses this functionality (currently OpenID and Auth0).

Is it really that bad to include also this in default requirements?

If yes, should I add _another_ requirements file, like `requirements-openidconnect.txt` and request it only on openshift implementation?

Reply:
For Azure it's up to user as well. There might be other social-auth backends with additional dependencies, we really can't depend on everything user might potentially set up here.

Note that I'm talking about dependencies in the code, it might make sense to include these in Docker or OpenShift integrations though.